src/Pages/sangwonma/Main/Comment.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import './Comment.scss';
+
+
+class Comment extends Component {
+    render() {
+        return (
+            <div>
+                <span>  {this.props.valueFromParent}</span>
+                <span> <img className="iconHeartSmall" alt="하트아이콘"
+                    src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/heart.png" /></span>
```

Comment:
```suggestion
                <span> 
                    <img 
                        className="iconHeartSmall" alt="하트아이콘"
                        src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/heart.png" />
                </span>
```

- 부모 자식 관계가 잘 드러날 수 있게 줄바꿈 처리 제대로 해주세요!